### PR TITLE
Publicize: add Bluesky and Threads to lists of supported networks

### PR DESCRIPTION
**Pull request by jeherve**
## Proposed changes:

Let's ensure our lists of supported social networks are up to date.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

See p1732022013890789-slack-C02JJ910CNL

## Does this pull request change what data or activity we track or use?
* No

## Testing instructions:

Not much to test here. You'll want to check for typos.

### CODE REVIEW

File: projects/plugins/social/readme.txt [hunk 1]
```diff
@@ -73,14 +73,16 @@ You can connect to any of the following networks:
...
+* Bluesky
 
 After you add a new connection, you have the option to mark the connection as shared, meaning it can also be used by any other users on your site who have the ability to publish posts.
 
 To make the connection available to all users, check the box labeled "Mark the connection as shared".
 
 = To which social media platforms can I post using Jetpack Social? =
 
-You can post to Facebook, Instagram Business, Tumblr, Mastodon, Linkedin and Nextdoor. We are working hard to increase our social share integrations.
+You can post to Facebook, Bluesky, Threads, Instagram Business, Tumblr, Mastodon, Linkedin, and Nextdoor. We are working hard to increase our social share integrations.
```
Comment (pfefferle) [inline projects/plugins/social/readme.txt:85]: Is it correct to have a `,` before an `and`?

File: projects/plugins/jetpack/readme.txt [hunk 2]
```diff
@@ -58,7 +58,7 @@ Get blazing fast site speed with Jetpack. Jetpack’s free CDN (content delivery
 = POWERFUL TOOLS FOR GROWTH =
 Create and customize your WordPress site, optimize it for visitors and revenue, and enjoy watching your stats tick up. Build it, share it, and watch it grow.
 
-* Auto publish blog posts and products to social media by simply using our tools to connect to Facebook, Tumblr, Mastodon, and LinkedIn.
+* Auto publish blog posts and products to social media by simply using our tools to connect to Facebook, Bluesky, Threads, Tumblr, Mastodon, LinkedIn, and Nextdoor.
```
Comment (pfefferle) [inline projects/plugins/jetpack/readme.txt:61]: Is it correct to have a `,` before an `and`?

**Review by pfefferle — APPROVED** [review]

File: projects/plugins/social/readme.txt (hunk 1 above)
Comment (jeherve) [reply projects/plugins/social/readme.txt:85]: Yay Oxford comma :)

File: projects/plugins/jetpack/readme.txt (hunk 2 above)
Comment (jeherve) [reply projects/plugins/jetpack/readme.txt:61]: Yay Oxford comma :)

**Pull request merged**